js/utils/sceneMap.js

```diff
@@ -59,6 +59,30 @@ const levels = [
         ['f', 'w', 'w', 'w', 'w', 'w', 'w', 'w', 'w', 'f'],
         ['f', 'f', 'f', 'f', 't', 'f', 'f', 'f', 'f', 'f']
     ],
+    [
+        ['f', 'f', 'f', 'f', 'f', 'f', 'f', 'f', 'f', 'd'],
+        ['f', 'f', 'f', 'f', 'f', 'f', 'f', 'f', 'f', 'f'],
+        ['f', 'f', 'f', 'f', 'f', 'w', 'f', 'f', 'f', 'f'],
+        ['f', 'f', 'f', 'f', 'f', 'w', 'f', 'f', 'f', 'f'],
+        ['f', 'f', 'w', 'h', 'f', 'w', 'f', 'f', 'f', 'f'],
+        ['w', 'w', 'w', 'w', 'w', 'w', 'f', 'f', 'f', 't'],
+        ['f', 'f', 'f', 'l', 'f', 'w', 'f', 'f', 't', 't'],
+        ['f', 'f', 'f', 'w', 'f', 'w', 'f', 'f', 'f', 'f'],
+        ['f', 'w', 'w', 'w', 'w', 'w', 't', 't', 't', 'f'],
+        ['f', 'f', 'f', 'f', 'f', 'f', 'f', 'f', 'f', 'f']
+    ],
+    [
+        ['h', 'w', 'f', 'f', 'f', 'w', 'f', 'f', 'f', 'f'],
+        ['f', 'w', 'f', 'w', 'f', 'w', 'f', 'f', 'f', 'f'],
+        ['f', 'w', 'f', 'w', 'f', 'w', 'f', 'f', 'f', 'f'],
+        ['f', 'w', 'f', 'w', 'f', 'w', 'f', 'f', 'f', 'f'],
+        ['f', 'w', 'f', 'd', 'f', 'w', 'l', 'f', 'f', 'f'],
+        ['f', 'w', 'f', 'w', 'f', 'w', 'f', 'f', 'f', 'f'],
+        ['f', 'w', 'f', 'w', 'f', 'w', 'f', 'f', 'f', 'f'],
+        ['f', 'w', 'f', 'w', 'f', 'w', 't', 't', 'f', 'f'],
+        ['f', 'w', 'f', 'w', 'f', 'w', 't', 'f', 'f', 'f'],
+        ['f', 'f', 'f', 'w', 'f', 'f', 'f', '', 'f', 'f']
```

Comment:
Is the empty string 3 from the right accidental?

Reply:
Absolutly not